static/js/tabs.js

```diff
@@ -79,5 +79,5 @@ $(function() {
   // If we're opening an adventure from the beginning (either through a link to /hedy/adventures or through a saved program for an adventure), we click on the relevant tab.
   // We click on `level` to load a program associated with level, if any.
   const adventureName = window.State && window.State.adventure_name;
-  $('[data-tab="t-' + (adventureName || 'level') + '"]').click ();
+  $('*[data-tab="t-' + (adventureName || 'level') + '"]').click ();
```

Comment:
Question: why do we need a wildcard here?

Reply:
Not sure. I was getting weird errors and adding this seemed to fix it. It might be a red herring.